src/ceph-volume/plugin/cache/ceph_volume_cache/main.py

```diff
@@ -0,0 +1,207 @@
+import argparse
+import sys
+from ceph_volume.api import lvm as api
+from ceph_volume.util import disk
+
+"""
+The user is responsible for splitting the disk into data and metadata partitions.
+
+If we are to partition a disk to be used as a cache layer, no partition can be
+smaller than 2GB because ceph-volume creates vgs with PE = 1GB.
+
+"""
+
+
+# Not very elegant, probably needs to be replaced or at least moved to util
+def strToSize(s):
```

Comment:
1) Maybe rename it to _sizeFromHumanReadable()_ and provide in inline documentation the string format it expects with explanation.
2) Since it is using disk.Size it is probably better to move this method to 'disk' module, and probable to add its usage to the disk.Size constructor via _hr_ parameter.
3) Add unit test coverage.
4) For if the suffix does not match either of T,G,M,K it should raise an exception instead of returning None?
5) Maybe better use regex?

Reply:
@kshtsk thanks a lot for your comments! I've addressed most of them. For this one, I really like the `Size(hr=s)` idea, I will implement it in the iteration. Will also add tests.